en/02_Developer_Guides/15_Customising_the_Admin_Interface/07_Status_Flags.md

```diff
@@ -0,0 +1,127 @@
+
+These flags are displayed at the top of edit forms in the CMS, and in various places next to record names such as the site tree and in grid fields.
+
+![status flags in the CMS](../../_images/status-flag.png)
+
+In the above screenshot, "MODIFIED" and "EN_US" are status flags. The "MODIFIED" flag is provided by the `Versioned` extension, and the "EN_US" flag is provided by [`tractorcow/silverstripe-fluent`](https://github.com/tractorcow-farm/silverstripe-fluent/).
+
+Note that in the site tree only one status flag will be displayed when you're editing a record, due to the limited space available.
+
+Some status flags have additional context which can be viewed by hovering over them with your mouse cursor.
+
+## Adding and removing status flags
+
+You can easily add and remove status flags.
+
+The customization of status flags can be done either directly in the model class by overriding the [`ModelData::getStatusFlags()`](api:SilverStripe\Model\ModelData::getStatusFlags()) method, or by [applying an extension](/developer_guides/extending/extensions/) that implements the `updateStatusFlags` extension hook.
+
+Add status flags by adding new items to the `$flags` array, and remove existing flags by unsetting the relevant key in the array.
+
+### Directly on the model
+
+```php
+namespace App\Model;
+
+use SilverStripe\ORM\DataObject;
+
+class MyModel extends DataObject
+{
+    // ...
+    private static array $db = [
+        'IsPublishScheduled' => 'Boolean',
+    ];
+
+    public function getStatusFlags(bool $cached = true): array
+    {
+        $this->beforeExtending('updateStatusFlags', function (array &$flags) {
+            if ($this->IsPublishScheduled) {
+                $flags['scheduledtopublish'] = _t(__CLASS__ . 'SCHEDULE_PUBLISH', 'Scheduled To publish');
+            }
+        });
+        return parent::getStatusFlags($cached);
+    }
+}
+```
+
+### Via an extension
+
+```php
+namespace App\Extension;
+
+use SilverStripe\Core\Extension;
+
+class SchedulePublishExtension extends Extension
+{
+    private static array $db = [
+        'IsPublishScheduled' => 'Boolean',
+    ];
+
+    protected function updateStatusFlags(array &$flags): void
+    {
+        if ($this->getOwner()->IsPublishScheduled) {
+            $flags['scheduledtopublish'] = _t(__CLASS__ . 'SCHEDULE_PUBLISH', 'Scheduled To publish');
+        }
+    }
+}
+```
+
+### Including additional context
+
+Sometimes the text of a status flag won't convey all the information you want it to, since it is so short.
+
+You can include additional context which will be displayed when hovering over your status flags. This is done using an associative array where the main text of the flag is in the `text` key, and the additional information is in the `title` key:
+
+```php
+namespace App\Model;
```

Comment:
I think this example should be changed like I've recommended above i.e. just use "MyColour" and don't use _t()